apps/server/src/modules/auth/next-auth.controller.ts

```diff
@@ -127,7 +131,7 @@ export class NextAuthController {
       } else if (action === 'signout') {
         res.send(JSON.stringify({ url: redirect }));
       } else {
-        res.redirect(redirect);
+        res.send(JSON.stringify({ url: redirect }));
```

Comment:
match the behavior with: https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/lib/routes/signin.ts#L52